src/main/java/svc/repositories/JudgeJpaRepository.java

```diff
@@ -0,0 +1,12 @@
+package svc.repositories;
```

Comment:
It's typically not ideal to name an interface to something technology specific. I'd remove the "JPA" part of these repository names.

Reply:
Why did you choose to extend from `JpaRepository` vs `CrudRepository`?

Reply:
See answer above.